### PR TITLE
Release/5.0.0-beta.2

### PR DESCRIPTION
**Pull request by MarkVillacampa**

### CODE REVIEW

**Comment by NachoSoto** [comment]
Maybe remove what's already coming out on #3644?

File: fastlane/Fastfile [hunk 1]
```diff
@@ -768,17 +768,14 @@ platform :ios do
...
               # and push the changes
               docs_destination_folder = "docs/#{version_number}"
-              index_destination_path = "docs/index.html"
-              migration_guide_destination_path = "docs/v4_api_migration_guide.html"
               FileUtils.cp_r docs_generation_folder + "/.", docs_destination_folder
-              FileUtils.cp docs_index_path, index_destination_path
-              FileUtils.cp docs_migration_guide_path, migration_guide_destination_path
+              docs_files.each do |file|
+                FileUtils.cp file, "docs/"
+              end
```
Comment (MarkVillacampa) [inline fastlane/Fastfile:774]: Added a new file here so toop the opportunity to generalize this to we don't need to add new files manually.

**Comment by NachoSoto** [comment]
Reminder to bring #3582, #3583, and #3615 if you can 🙏🏻

**Comment by MarkVillacampa** [comment]
Rebased on top of https://github.com/RevenueCat/purchases-ios/tree/5.0-dev

**Comment by MarkVillacampa** [comment]
This has been released.

Created https://github.com/RevenueCat/purchases-ios/pull/3658 to add the changelog + migration guide to the `5.0-dev` branch.

Closing this instead of merging to avoid future merge conflicts with the version change commits.

**Pull request closed without merging**